### ISSUE TITLE
DNS error for Search API

### ISSUE THREAD

**Issue by aleksandergrzyb** [CocoaPods/CocoaPods#10051]
👋, is [Search API](https://blog.cocoapods.org/Search-API-Version-1/) still maintained? 🤔 For me currently it results with DNS error.

**Comment by dnkoutso** [comment]
works for me. just tried `pod search Google` and got results.

Try `--verbose` and if possible please provide more information to diagnose.

**Comment by aleksandergrzyb** [comment]
True, but when I go to the address http://search.cocoapods.org/api/pods it gives me DNS error, or am I doing something wrong? 🤔

**Comment by dnkoutso** [comment]
nope. i am not sure why this doesnt work. Transferring issue though to cocoapods.org
